lib/archethic/p2p.ex

```diff
@@ -675,22 +692,60 @@ defmodule Archethic.P2P do
           rest,
           message,
           conflict_resolver,
+          acceptance_resolver,
           consistency_level,
           timeout,
           previous_result
         )
 
       1 ->
         if previous_result != nil do
-          do_quorum([previous_result | results], conflict_resolver)
+          quorum_result = do_quorum([previous_result | results], conflict_resolver)
+
+          if acceptance_resolver.(quorum_result) do
+            {:ok, quorum_result}
+          else
+            do_quorum_read(
+              rest,
+              message,
+              conflict_resolver,
+              acceptance_resolver,
+              consistency_level - 1,
```

Comment:
I think you should not reduce the consistency level as we want at least 3 results (by defaut), so here only 2 nodes will be asked instead of 3. Then 1, then 0.

Reply:
ty